src/modules/logger/logger.cpp

```diff
@@ -575,53 +575,60 @@ void Logger::add_common_topics()
 	// Note: try to avoid setting the interval where possible, as it increases RAM usage
 	add_topic("actuator_controls_0", 100);
 	add_topic("actuator_controls_1", 100);
-	add_topic("actuator_outputs", 100);
-	add_topic("airspeed");
+	add_topic("actuator_outputs", 200);
+	add_topic("airspeed", 200);
 	add_topic("att_pos_mocap", 50);
-	add_topic("battery_status", 300);
+	add_topic("battery_status", 500);
 	add_topic("camera_capture");
 	add_topic("camera_trigger");
-	add_topic("commander_state", 100);
+	add_topic("commander_state", 200);
 	add_topic("control_state", 100);
 	add_topic("cpuload");
-	add_topic("differential_pressure", 50);
-	add_topic("distance_sensor");
-	add_topic("ekf2_innovations", 50);
-	add_topic("esc_status", 250);
-	add_topic("estimator_status", 200); //this one is large
-	add_topic("gps_dump"); //this will only be published if gps_dump_comm is set
```

Comment:
I'm not a fan of commented code like that. It would be pretty easy to add as another logger profile and then accessible via param. What's the use case for gps_dump?

gps_dump + satellite_info as "GPS extra"?